doc/TBG_macros.cfg

```diff
@@ -196,6 +196,8 @@ TBG_liveViewYX="--live_e.period 1 --live_e.slicePoint 0.5 --live_e.ip 10.0.2.254
 TBG_liveViewYZ="--live_e.period 1 --live_e.slicePoint 0.5 --live_e.ip 10.0.2.254 \
                 --live_e.port 2021 --live_e.axis yz"
 
+# print the maximum charge deviation between particles and div E
+TBG_chargeConservation="--chargeConservation_period 100"
```

Comment:
pls use `chargeConservation.period` in the plugin (as all other plugins should do)